.gitignore

```diff
@@ -2,3 +2,4 @@ elm-stuff
 elm.js
 index.html
 node_modules
+yarn.lock
```

Comment:
Shouldn't we be checking this file in? It will further reduce chances of improper versions of node packages.